src/dune

```diff
@@ -0,0 +1,9 @@
+(library
+  (name opam_file_format)
+  (public_name opam-file-format)
+  (synopsis "opam file format handling library")
```

Comment:
This should be `"Parser and printer for the opam file syntax"`

Reply:
fixed in 43bbd4831564a881a5f31dd4d4fbe8c2bbd13f39